core/src/commonMain/kotlin/com/xebia/functional/xef/conversation/Conversation.kt

```diff
@@ -90,14 +90,15 @@ constructor(
   ): Flow<StreamedFunction<A>> = chat.promptStreaming(prompt, this@Conversation, serializer())
 
   @AiDsl
-  fun promptStreaming(
+  suspend fun promptStreaming(
```

Comment:
Why is this `suspend` needed? That should be captured in the `Flow`, no?

Reply:
not needed, thanks!